prog/vm/nexus.rb

```diff
@@ -437,24 +437,21 @@ def before_run
   end
 
   label def wait_sshable
-    addr = vm.ephemeral_net4
-
-    # Alas, our hosting environment, for now, doesn't support IPv6, so
-    # only check SSH availability when IPv4 is available: a
-    # unistacked IPv6 server will not be checked.
-    #
-    # I considered removing wait_sshable altogether, but (very)
-    # occasionally helps us glean interesting information about boot
-    # problems.
-    hop_create_billing_record unless addr
+    case host.sshable.cmd("common/bin/daemonizer --check wait_sshable_#{q_vm}")
+    when "Succeeded"
+      vm.vm_host.sshable.cmd("common/bin/daemonizer --clean wait_sshable_#{q_vm}")
```

Comment:
We use both `host` and `vm.vm_host`. We can unify them.
```suggestion
    case host.sshable.cmd("common/bin/daemonizer --check wait_sshable_#{q_vm}")
    when "Succeeded"
      host.sshable.cmd("common/bin/daemonizer --clean wait_sshable_#{q_vm}")
```